cadc-vosi/src/main/java/ca/nrc/cadc/vosi/avail/CheckDataSource.java

```diff
@@ -168,6 +188,20 @@ public void check()
         }
         finally
         {
+            if (this.rollback && con != null)
+            {
+                try
+                {
+                    con.rollback();
+                    con.setAutoCommit(true);
+                }
+                catch (SQLException e)
+                {
+                    log.debug("rollback failed: " + dataSourceName + " (" + testSQL + ")", e);
+                    log.warn("rollback failed: " + dataSourceName + " (" + testSQL + ")");
```

Comment:
Sure. I can remove 'log.warn...'. I was just following the 'catch (SQLException e) {...} above. Maybe we should remove the 'log.warn ...' there as well.